css/style.css

```
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+    /* свойство scroll-behavior не наследуется, применяется к прокручиваемым блокам */
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+    display: inline-block;
+}
+
+.header {
+    font-family: Montserrat, sans-serif;
+    height: 806px;
+    background-color: #eff0f2;
+}
+
+.container {
+    width: 1140px;
+    margin: 0 auto;
+}
+
+.header_logo {
+    margin-left: 0%;
+    padding-top: 5.7%;
+}
+
+.header_text {
+    margin-top: 16.7%;
+    margin-left: 5%;
+    text-align: center;
+    margin-bottom: 10%;
+}
+
+.header_text__style {
+    margin-top: 200px;
+    margin-bottom: 137px;
+    text-align: center;
+    font-size: 45px;
+    line-height: 59px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    text-shadow: 4px 8px 10px rgb(0 0 0 / 23%);
+    color: #505050;
+}
+
+.header_text__style span {
+    background-color: #76f4ea;
+    padding: 0 15px;
+}
+
+.header_button {
+    display: flex;
+    justify-content: center;
+    /* padding-top: 12%; */
+    margin-left: 0%;
+}
+
+.header_button__style {
+    display: block;
+    margin: 0 auto;
+    width: 232px;
+    height: 53px;
+    position: relative;
+    font-size: 18px;
+    line-height: 50px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    color: #000;
+    border: 2px solid #000;
+    border-radius: 2px;
+    text-align: center;
+    transition: .3s;
+}
+
+.header_button__style:hover {
+    cursor: pointer;
+    background-color: rgb(103, 189, 192);
+    transition: .3s;
+    transform: scale(1.1);
+}
+
+.header_button__text {
+    font-family: Montserrat;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.12em;
+    color: #000000;
+    scroll-behavior: smooth;
+}
+
+.header_image {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 5%;
+    margin-top: -3.1%;
+}
+
+.main-content {
+    padding-top: 10.7%;
+    min-height: 985px;
+    background-color: #f8fafa;
+}
+
+.main-intro {
+    text-align: center;
+    margin-left: 23px;
+}
+
+.main-information {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 141px;
+}
+
+.main-intro__text-par {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.043em;
+    color: #000000;
+}
+
+.main-intro__text-content {
+    opacity: 0.61;
+    font-family: Merriweather;

```

use css variables for `font-family`

---

tnanks ! 
i fixed it
```suggestion
 font-family: var(--font-Merri);
```